### DISCUSSION TITLE
How to scale two displays differently?

### DISCUSSION THREAD

**Discussion by deadwax** [lxqt/lxqt#2526] [Q&A] [answered]
Hi, I'm using LXQt under Debian testing, xfwm4 WM, on a surface pro, which has a high resolution small internal monitor (2736x1824) and a lower resolution larger external monitor (1920x1200).  With things set at 96dpi the external monitor looks normal and usable and the internal monitor is unusable - everything is much too small and essentially impossible to read.  Flipping things about looks no more sensible.

How can I achieve things looking a sane size on both monitors?

**Comment by stefonarch** [comment]
I'm really  not expert of dual monitor setups, but when I look at `wdisplays` on lawbc it looks like wayland enables different scalings:

![screen_area_dom_17:48:59_](https://github.com/lxqt/lxqt/assets/10681413/8724e189-4056-48d9-ac76-7b0060464c30)

I'm using for months https://github.com/stefonarch/lxqt-labwc-session/ now on debian 12 and arch, it has it's limitations though.

**Comment by stefonarch** [comment, marked as answer]
Can be easily configured using any Wayland compositor.
